storage.go

```diff
@@ -252,14 +290,8 @@ func (s *Storage) writeAppend(ctx context.Context, o *Object, r io.Reader, size
 
 	options := make([]oss.Option, 0)
 	options = append(options, oss.ContentLength(size))
-	if opt.HasServerSideEncryption {
-		options = append(options, oss.ServerSideEncryption(opt.ServerSideEncryption))
-	}
-	if opt.HasServerSideDataEncryption {
-		options = append(options, oss.ServerSideDataEncryption(opt.ServerSideDataEncryption))
-	}
-	if opt.HasServerSideEncryptionKeyID {
-		options = append(options, oss.ServerSideEncryptionKeyID(opt.ServerSideEncryptionKeyID))
+	if opt.HasContentMd5 {
```

Comment:
We should pass the `content_type` the first time of the `AppendObject` call.

As we added the `content_type` in Object, but that value is not stored by service. If the user tries to `Stat` an object, he/she will found those metadata is missing.

Reply:
`content-type` has set in `create_append`.